### DISCUSSION TITLE
[Feature]: scrubbable scroll timeline and lazy loading in album view

### DISCUSSION THREAD

**Discussion by JeromeGillard** [immich-app/immich#1622] [Feature Request]
### Feature detail

It would be nice to improve the load time of large libraries by using paginated requests and infinite scroll.

For instance, viewing the content of my album with about 19k items generates a 19 MB JSON payload. Too painful to generate (server side), load (network), and interpret (client side).

Affected path for the web version: 
 - the `/photos` screen (`/api/asset/time-bucket`)
 - the `/albums/<uuid>` screen (`/albums/<uuid>/__data.json`)

### Platform

Web

**Comment by alextran1502** [comment]
We have plan to integrate the scrubbable scroll view of the main timeline to album view as well. This is not a high priority because usually the number of assets are pretty low comparing to the library with the exception of yours

**Reply by mirisbowring** [reply]
I also have the Problem with a "small" album of around 600 Images. It takes around 20 Seconds to finally load the album.

**Comment by jrasm91** [comment]
Tracking in #2337
